cherryontech/do-five-things
# chore: Add storybook

## Context
Devs who want to see their components as they develop.

## Work done
Added storybook, with an example story for Settings Inputs.

## Manual testing 
0. Make sure you yarn at the root so you can get the new packages.
1. run `yarn storybook`.
2. Navigate to the inputs page (listed in the sidebar).
- Expect that the component shows.
- Expect the controls in SB to be functional.
- Expect not to see any errors or warnings in the console (at least, none related to this component).
